### DISCUSSION TITLE
Adding to NextJS 13.4.9 with app routing

### DISCUSSION THREAD

**Discussion by nelsonmandeladev** [jellydn/next-swagger-doc#769] [General]
Hi there,
Please I wanted to know if  **next-swagger-doc**  can be used with NextJS 13.4.9^ with app routing. If yes, is there any reference?

**Comment by jellydn** [comment]
Hi @nelsonmandeladev Do you check the example apps folder on this repo?

**Reply by nelsonmandeladev** [reply]
Hi @jellydn, I did not find it that's why I asked for any reference. Can you please just reference me there?

**Reply by jellydn** [reply]
https://github.com/jellydn/next-swagger-doc/tree/main/examples/next13-app

**Reply by nelsonmandeladev** [reply]
Thank very @jellydn
